plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -4175,13 +4250,18 @@ public void registerCoreMembers() {
         // @Group npc
         //
         // @Description
-        // TODO: Document Command Details
+        // Toggles whether a NPC can be hurt or not. (Defaults to false)
+        // If you specify a state, sets the NPC to that.

```

Comment:
`Toggles whether an NPC can be hurt. (Defaults to false)` is probably sufficient for this description